org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java

```diff
@@ -241,6 +241,15 @@ protected void assertPost(String restUrl, Path file) {
 			.statusCode(201);
 	}
 
+	protected void assertPostWithOverwrite(String restUrl, Path file, boolean overwrite) {
```

Comment:
Move to test class; if possible. This thing is required only once. you can make `start()` protected.

Reply:
Please really remove the test method here 😇

Reply:
Sorry, forgot to do this. Will be done in a few minutes.